backend/src/main/java/com/capstone/moneytree/facade/MarketInteractionsFacade.java

```diff
@@ -188,10 +196,9 @@ public void onStreamUpdate(AlpacaStreamMessageType streamMessageType, AlpacaStre
                TradeUpdateMessage tradeMessage = (TradeUpdateMessage) streamMessage;
                TradeUpdate tradeUpdate = tradeMessage.getData();
                if (tradeUpdate.getEvent().equals("fill")) {
-                  messageSender.convertAndSend(
-                          "/queue/user-" + userId,
-                          tradeUpdate.getOrder().getClientOrderId());
+                  messageSender.convertAndSend("/queue/user-" + userId, tradeUpdate.getOrder());
                   sendOrderCompletedEmail(userId, tradeUpdate);
```

Comment:
This is where we should update the transaction no ?, it is taken care of in another Pr right ? (refactoring PR ?)

Reply:
Yes it is taken care of in another PR.